linode/ipv6range/framework_datasource.go

```diff
@@ -41,6 +45,9 @@ func (d *DataSource) Read(
 	rangeStrSplit := strings.Split(data.Range.ValueString(), "/")
 	rangeStr := rangeStrSplit[0]
 
+	ctx = tflog.SetField(ctx, "ipv6_range", data.Range.ValueString())
```

Comment:
Do you think it will be better to log `rangeStr` instead of `data.Range.ValueString()`, since it's what we actually send to the API to get the ipv6 range?